.github/workflows/pypi-publish.yml

```diff
@@ -0,0 +1,59 @@
+---
+name: Publish Python Package
+
+on:
+  release:
+    types: [created]
```

Comment:
```suggestion
    types: [published]
```
**suggestion:** use same event as in modos-api for consistency (I believe the difference is that "created" is also triggered by drafts)